scripts/globals/items/bottle_of_monarchs_drink.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------------
+-- ID: 5393
+-- Item: Bottle of Monarch's Drink
+-- Item Effect: Regain 30TP/sec 180
+-- Duration 3 Mins
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+	if target:hasStatusEffect(dsp.effect.MEDICINE) then
```

Comment:
this does not.. 

Ok look, re-review ALLL your tempt items alright? They just do not medicate. Temp items are special.

Reply:
Done